common/components/forms/PositionList.jsx

```diff
@@ -129,40 +211,48 @@ class PositionList extends React.PureComponent<Props, State> {
           message="Do you want to delete this position?"
```

Comment:
Is it easy to add the role name to the modal text so it's clearer which role is being deleted? If yes, ask design if they like that idea?

I noticed in testing if I had a list of 5 roles and I clicked delete the modal would come up but which specific delete I clicked on was not clear if I moved my mouse.

Reply:
Also agree, but it's enough of a lift given the way the code is organized I'd prefer to get design input first.

Reply:
As above, sounds good.